pkg/common-controller/framework_test.go

```diff
@@ -1053,6 +1490,64 @@ func newSnapshot(
 	return &snapshot
 }
 
+func newGroupSnapshot(
+	snapshotName, snapshotUID string, selectors map[string]string, targetContentName, snapshotClassName, boundContentName string,
```

Comment:
`snapshotName` to `groupSnapshotName` `snapshotUID` to `groupSnapshotUID` `snapshotClassName` to `groupSnapshotClassName`

Reply:
Fixed!